src/components/NavBar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './NavBar.css';
+import logo from '../assets/icon.webp';
+
+const NavBar = () => (
+  <nav className="nav-bar">
+    <p>
+      <img alt="LOGO" src={logo} />
+      Math Magicians
+    </p>
+    <ul className="links-nav">
+      <li><a href="/"> Home </a></li>
+      <li><a href="/calculator"> Calculator </a></li>
+      <li><a href="/quote"> Quote </a></li>
```

Comment:
- [ ] Please add a `<Link>` tag instead of the `<a>` to complete the setup of `react-router` links for the routes you created in the `App.js` module So your app can act as a Single Page App. (SPA). 
[Hint]: Kindly see that [link ](https://reactrouter.com/docs/en/v6/getting-started/tutorial#add-some-links)to know how to do it properly.

### [P.S.] This is the most beautiful design I ever reviewed for this project even way better than my design, So Keep up the good work 💪.

Reply:
Hi @Omar-Muhamad , thanks a lot for your review. I completely forgot about the links!
I think you're the first one I got that actually commented something nice about the project when it's basically done. Most reviewers just approve and don't mention anything. So, thanks a lot again for the compliment, it's nice when we get acknowledge about something we put a lot of effort on!

Have a great rest of the week!